src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordClone.cs

```diff
@@ -17,11 +17,25 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SynthesizedRecordClone : SynthesizedInstanceMethodSymbol
     {
+        public override TypeWithAnnotations ReturnTypeWithAnnotations { get; }
         public override NamedTypeSymbol ContainingType { get; }
+        public override bool IsOverride { get; }
```

Comment:
>IsOverride [](start = 29, length = 10)

It looks like the Clone method is now always called virtually, we should reflect that in CFG. If any additional fix-ups will be necessary for this PR, consider adjusting this [https://github.com/dotnet/roslyn/pull/44712#pullrequestreview-425708407](https://github.com/dotnet/roslyn/pull/44712#pullrequestreview-425708407)